src/main/java/org/cactoos/scalar/ScalarOf.java

```diff
@@ -23,32 +23,61 @@
  */
 package org.cactoos.scalar;
 
+import org.cactoos.Func;
+import org.cactoos.Proc;
 import org.cactoos.Scalar;
+import org.cactoos.func.FuncOf;
 
 /**
  * ScalarOf.
  *
+ * @param <X> Element type
```

Comment:
@fabriciofx This type parameter should be on ctor level.

Reply:
@andreoss Can you explain it better? Shoul I add `X` to ctor, as ` public <X> ScalarOf(final Runnable runnable, final T result)`?

Reply:
@fabriciofx Exactly, it should like that
```
    public X<X> ScalarOf(final Func<? super X, ? extends T> fnc, final X ipt) {
```

Reply:
@andreoss done, and thanks for the help! :)